PsUtil/config.py

```diff
@@ -22,14 +22,30 @@
 
 	"commands" : [
 
-		"{buildDir}/bin/python setup.py install --root / --prefix {buildDir}",
+		"{buildDir}/bin/python setup.py install --root / --prefix {buildDir} {extraArgs}",
 
 	],
 
+	"variables" : {
+
+		"extraArgs" : "",
+
+	},
+
 	"manifest" : [
 
 		"lib/python{pythonVersion}/site-packages/psutil",
 
 	],
 
+	"platform:macos" : {
+
+		"variables" : {
+
+			"extraArgs" : "--install-lib {pythonLibDir}/python{pythonVersion}/site-packages",
```

Comment:
Waaay back when, I was trying to get all the Python modules installing into `{buildDir}/python`. I gave up on that eventually, which leaves us with a mishmash of that and `site-packages`. Do you think we should just have everything install to `site-packages` now, including Gaffer's own modules?

Reply:
I can see the potential cleanliness of putting everything under `site-packages`, but on the other hand I think I'd prefer the mental separation of Gaffer's modules under `{buildDir}/python` and third-party in `site-packages`. Right now we're at neither of those ideals as `{buildDir}/python` is still a mish-mash of Gaffer/Cortex plus Imath, MaterialX, USD, etc.

All I've done here is attempt to corral the stuff that was trying to install to the incorrect `site-packages` path on macOS, but I wouldn't mind taking another pass at this later to tidy things up generally in one of the above directions or the other...